base/shell/explorer/traywnd.cpp

```diff
@@ -1628,14 +1628,21 @@ class CTrayWindow :
         if (StartSize.cx > rcClient.right)
             StartSize.cx = rcClient.right;
 
-        if (!m_Theme)
+        HWND hwndTaskToolbar = ::GetWindow(m_TaskSwitch, GW_CHILD);
+        if (hwndTaskToolbar)
         {
-            HWND hwndTaskToolbar = ::GetWindow(m_TaskSwitch, GW_CHILD);
-            if (hwndTaskToolbar)
+            SIZE EdgeSize;
+            EdgeSize.cy = GetSystemMetrics(SM_CYEDGE);
+            DWORD size = SendMessageW(hwndTaskToolbar, TB_GETBUTTONSIZE, 0, 0);
+            if (!m_Theme)
             {
-                DWORD size = SendMessageW(hwndTaskToolbar, TB_GETBUTTONSIZE, 0, 0);
                 StartSize.cy = HIWORD(size);
             }
+            else
+            {
+                /* Themed button covers Edge area as well */
+                StartSize.cy = HIWORD(size) + EdgeSize.cy;
+            }
```

Comment:
Seems that all the code from the `if (!m_Theme)` above down to here could be simplified by:
```c
            /* Themed button covers Edge area as well */
           StartSize.cy = HIWORD(size) + (m_Theme ? GetSystemMetrics(SM_CYEDGE) : 0);
```

Reply:
I tried this and now I have a strange build failure. Let me try this again.

Reply:
I had an extra parenthesis. Now this should be good. Thanks for your suggestions @HBelusca.